plugins-base/XSFeatureSRCommon.py

```diff
@@ -226,7 +226,7 @@ def UpdateFieldsCONFIRM(self):
         pane.ResetFields()
 
         sr = HotAccessor().sr[self.srHandle]
-        srName = sr.name_label(None).encode('utf-8')
+        srName = Lang(sr.name_label(None))
```

Comment:
Why've you used `Lang()` here and elsewhere in the commit? I think it is not normally used with dynamic values. Normally it is only used with static labels like `Operation`, `Storage Repository`, etc.

Reply:
I used `Lang()` as I'm not sure the object is a string or a bytes object. Let's remove `encode` here to see how it goes.